libraries/AP_BattMonitor/AP_BattMonitor_FuelLevel_Analog.cpp

```diff
@@ -115,6 +159,8 @@ void AP_BattMonitor_FuelLevel_Analog::read()
     // map consumed_mah to consumed millilitres
     _state.consumed_mah = fuel_level_used_ratio * _params._pack_capacity;
 
+    _state.current_amps = voltage;   
```

Comment:
current == fuel level? that is odd
maybe better as zero?

Reply:
I think it would be better to make has_current() in AP_BatteryMonitor_FuelLevel_Analog.h return false

Reply:
Done